### PR TITLE
Video schema

### PR DESCRIPTION
**Pull request by chethus**
Defines the video schema and tests it using an input form and VideoServlet. After entering a video ID, the Video object JSON is returned.
Input Form:
Screenshot 2020-06-25 at 4 31 23 PM
Response:
Screenshot 2020-06-25 at 4 34 18 PM

### CODE REVIEW

**Comment by QuinnDACollins** [comment]
Nikhil might NIT some of the white space between import statements, otherwise LGTM

**Review by QuinnDACollins — APPROVED** [review]

**Comment by chethus** [comment]
> Nikhil might NIT some of the white space between import statements, otherwise LGTM

Yeah I just wanted to separate my own classes from imported classes, but maybe it's unnecessary.

File: portfolio/src/main/java/com/google/sps/data/Fields.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package com.google.sps.data;
+
+// Stores field names as constants.
+// Prevents errors due to typos when writing a field name string.
+public class Fields {
```
Comment (nikhilbirmiwal) [inline portfolio/src/main/java/com/google/sps/data/Fields.java:5]: NIT: I would name this RequestParameters, since that's where it's being used.

**Review by nikhilbirmiwal — APPROVED** [review]

File: portfolio/src/main/java/com/google/sps/data/Fields.java (hunk 1 above)
Comment (chethus) [reply portfolio/src/main/java/com/google/sps/data/Fields.java:5]: Whoops, I didn't see this before I merged. I think I'll have to change it in my next PR (or someone else can).

**Pull request merged**